pkg/reconciler/integration/testdata/defaults/serving_v1alpha1.yaml

```diff
@@ -0,0 +1,27 @@
+# This file is used to assert that the defaulting for serving doesn't change.
```

Comment:
WDYT about adding a README.md somewhere describing how to add new tests?

Reply:
added one here and in the integration directory